src/components/Search/Filters/TelehealthInput.tsx

```diff
@@ -0,0 +1,56 @@
+import { Fieldset } from "@trussworks/react-uswds";
+import { Dispatch, SetStateAction } from "react";
+import { useTranslation } from "react-i18next";
+import { SearchFilters, Telehealth } from "../../../types";
+import { AnalyticsAction, logEvent } from "../../../utils/analytics";
+import FilterRadio from "./FilterRadio";
+
+type TelehealthInputProps = {
+  legend: string;
+  compact?: boolean;
+  filters: SearchFilters;
+  setFilters: Dispatch<SetStateAction<SearchFilters>>;
+};
+
+function TelehealthInput({
+  legend,
+  compact = false,
+  filters,
+  setFilters,
+}: TelehealthInputProps) {
+  const { t } = useTranslation();
+
+  const setTelehealthFilter = (telehealth: Telehealth) => {
+    logEvent(AnalyticsAction.UpdateFilter, {
+      label: "telehealth",
+      filter_type: "telehealth",
+      filter_value: telehealth,
+    });
+
+    setFilters({
+      ...filters,
+      telehealth,
+    });
+  };
+
+  return (
+    <Fieldset legend={legend} legendStyle={compact ? "srOnly" : "large"}>
+      {[
+        Telehealth.InPersonAndTelehealth,
+        Telehealth.InPersonOnly,
+        Telehealth.TelehealthOnly,
+      ].map((option) => (
+        <FilterRadio
+          name={`telehealth-${compact ? "desktop" : "mobile"}`}
```

Comment:
Does the `name` prop have to be translated as well?

Reply:
As i understand, the `name property is for use by form logic (e.g. to group radio buttons), where as the `label` property is what is relied on for user-facing labelling etc.